tofhir-server/src/main/scala/io/tofhir/server/model/MappingMetadata.scala

```diff
@@ -0,0 +1,9 @@
+package io.tofhir.server.model
+
+/**
+ * Mapping model in the projects.json file
+ * @param id Identifier of the mapping
+ * @param url URL of the mapping
+ * @param name Name of the mapping file
+ */
+case class MappingMetadata(id: String, url: String, name: String)
```

Comment:
We already have a MappingDefinition class, but is it because we have FhirMappingExpression sequence in that class, that you needed a standalone metadata class?

Reply:
If we must create such a metadata class, I would name it FhirMappingMetadata. But first, let's check whether we can use FhirMapping by converting the non-metadata fields to Option or not.

Reply:
Actually, not because of just FhirMappingExpression sequence but also other fields (source, context, variable). We dont want those fields in the projects.json file. If we change those fields to Option, many in the engine codebase will be effected.

Reply:
OK. I recommend to rename it to FhirMappingMetadata and write our reasoning for its creation as a comment at the beginning of class definition.